### ISSUE TITLE
audio/webm detected as video/webm

### ISSUE THREAD

**Issue by EdouardLauret** [sindresorhus/file-type#488]
Hi,

I recorded a slice of audio using the MediaRecorder API.
```javascript
new MediaRecorder(myStream, {
    mimeType: 'audio/webm;codecs=opus',
    bitsPerSecond: 128000
});
```

When I try to detect its mime type with file-type it returns 
```javascript
{
   ext:'webm',
   mime: 'video/webm'
}
```

does anyone have an idea to overcome this problem ? 

Thank you in advance.

**Comment by EdouardLauret** [comment]
Nevermind, found #271.

**Comment by Borewit** [comment]
@EdouardLauret #271 is outdated.

Duplicate of issue #485 and resolved in #486. Yet the fix is not released yet.
